components/contextMenu/contextMenu.tsx

```diff
@@ -164,12 +164,43 @@ export default function CustomContextMenu(props: Props) {
     [ticketDetail],
   );
 
+  /*
+   * @desc update ticket snooze
+   */
+  const handleChangeSnooze = useCallback(
+    async (props: HandleClickProps) => {
+      const { item } = props;
+      const payload = { status: TicketStatus.OPEN, snoozeUntil: item?.value };
+      try {
+        if (ticketDetail?.id) {
+          const updatedTicketDetails = {
+            ...(ticketDetail || {}),
+            status: TicketStatus.OPEN,
+            snooze_until: new Date(item?.value || ''),
+          };
+          // add data in mobX store
+          ticketStore.setTicketDetails(updatedTicketDetails);
+          // api call for change ticket status
+          await changeTicketStatus(ticketDetail?.id, payload);
+        }
+      } catch (e) {
+        console.log('Error : ', e);
+      }
+    },
+    [ticketDetail],
+  );
+
   const [showDropDown, setShowDropDown] = useState(true);
   const [showDatePicker, setShowDatePicker] = useState(false);
 
-  const handleDropDownChange = () => {
-    setShowDropDown(false);
-    setShowDatePicker(true);
+  const handleDropDownChange = (item: any) => {
```

Comment:
Can you use here type properly
if you not found proper type use item: {name:string}